### DISCUSSION TITLE
Welcome to python-cheatsheet Discussions!

### DISCUSSION THREAD

**Discussion by wilfredinni** [wilfredinni/python-cheatsheet#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mithun162001** [comment]
Really good one!!! Keep it up...

**Comment by moisesrp21** [comment]
This is awesome.

**Reply by wilfredinni** [reply]
Thank you!

**Comment by KK4HFJ** [comment]
I found this recently when preparing for a python tech interview.  Was very helpful. I'm doing a lot of API work, as well as cisco updates using the python telnet library. hope to contribute soon.

**Comment by noraa-july-stoke** [comment]
Hi, I followed the invitation to contribute but can't figure out what is making the checker to open a pull request?

**Reply by noraa-july-stoke** [reply]
Also, if I did it in the wrong branch I apologize

**Comment by siphosethuraga** [comment]
Hi, I'm a n00b here would like to know the ins and outs of the software engineering... I need the help I can get...trust me I'm not weird...i mean, sometimes before I go yo bed...🤔 I do forget to brush my teeth...but it's nothing major....
